Broken gforker builds in mpi4py's GitHub actions

I believe the problem started after merging #7688

All the gforker builds fail to link the `mpiexec` executable
https://github.com/mpi4py/mpi4py-testing/actions/runs/22009561998
For example:
```
 CCLD     src/pm/gforker/mpiexec
/usr/bin/ld: src/pm/gforker/src_pm_gforker_mpiexec-mpiexec.o: in function `main':
/home/runner/work/mpi4py-testing/mpi4py-testing/mpi-build/src/pm/gforker/mpiexec.c:167:(.text+0x176): undefined reference to `MPL_trmalloc'
/usr/bin/ld: src/pm/gforker/src_pm_gforker_mpiexec-mpiexec.o: in function `mpiexec_usage':
/home/runner/work/mpi4py-testing/mpi4py-testing/mpi-build/src/pm/gforker/mpiexec.c:280:(.text+0x4f6): undefined reference to `MPL_usage_printf'
/usr/bin/ld: src/pm/gforker/src_pm_gforker_mpiexec-mpiexec.o: in function `mypreamble':
/home/runner/work/mpi4py-testing/mpi4py-testing/mpi-build/src/pm/gforker/mpiexec.c:311:(.text+0x5e9): undefined reference to `MPL_strnapp'
```

--- a/src/pm/gforker/Makefile.mk
+++ b/src/pm/gforker/Makefile.mk
@@ -14,17 +14,15 @@ if BUILD_PM_GFORKER
 if PRIMARY_PM_GFORKER
 bin_PROGRAMS += src/pm/gforker/mpiexec
 src_pm_gforker_mpiexec_SOURCES = src/pm/gforker/mpiexec.c 
-src_pm_gforker_mpiexec_LDADD = src/pm/util/libmpiexec.la $(mpllib)
-src_pm_gforker_mpiexec_LDFLAGS = $(mpllibdir)
-EXTRA_src_pm_gforker_mpiexec_DEPENDENCIES = $(mpllib)
+src_pm_gforker_mpiexec_LDADD = src/pm/util/libmpiexec.la $(mpl_lib)
+EXTRA_src_pm_gforker_mpiexec_DEPENDENCIES = $(mpl_lib)
 # we may not want to add AM_CPPFLAGS for this program
 src_pm_gforker_mpiexec_CPPFLAGS = $(common_pm_includes) $(AM_CPPFLAGS)
 else !PRIMARY_PM_GFORKER
 bin_PROGRAMS += src/pm/gforker/mpiexec.gforker
 src_pm_gforker_mpiexec_gforker_SOURCES = src/pm/gforker/mpiexec.c 
-src_pm_gforker_mpiexec_gforker_LDADD = src/pm/util/libmpiexec.la $(mpllib)
-src_pm_gforker_mpiexec_gforker_LDFLAGS = $(mpllibdir)
-EXTRA_src_pm_gforker_mpiexec_gforker_DEPENDENCIES = $(mpllib)
+src_pm_gforker_mpiexec_gforker_LDADD = src/pm/util/libmpiexec.la $(mpl_lib)
+EXTRA_src_pm_gforker_mpiexec_gforker_DEPENDENCIES = $(mpl_lib)
 # we may not want to add AM_CPPFLAGS for this program
 src_pm_gforker_mpiexec_gforker_CPPFLAGS = $(common_pm_includes) $(AM_CPPFLAGS)
 endif !PRIMARY_PM_GFORKER

--- a/src/pm/remshell/Makefile.mk
+++ b/src/pm/remshell/Makefile.mk
@@ -14,17 +14,15 @@ if BUILD_PM_REMSHELL
 if PRIMARY_PM_REMSHELL
 bin_PROGRAMS += src/pm/remshell/mpiexec
 src_pm_remshell_mpiexec_SOURCES = src/pm/remshell/mpiexec.c 
-src_pm_remshell_mpiexec_LDADD = src/pm/util/libmpiexec.la $(mpllib)
-src_pm_remshell_mpiexec_LDFLAGS = $(mpllibdir)
-EXTRA_src_pm_remshell_mpiexec_DEPENDENCIES = $(mpllib)
+src_pm_remshell_mpiexec_LDADD = src/pm/util/libmpiexec.la $(mpl_lib)
+EXTRA_src_pm_remshell_mpiexec_DEPENDENCIES = $(mpl_lib)
 # we may not want to add AM_CPPFLAGS for this program
 src_pm_remshell_mpiexec_CPPFLAGS = $(common_pm_includes) $(AM_CPPFLAGS)
 else !PRIMARY_PM_REMSHELL
 bin_PROGRAMS += src/pm/remshell/mpiexec.remshell
 src_pm_remshell_mpiexec_remshell_SOURCES = src/pm/remshell/mpiexec.c 
-src_pm_remshell_mpiexec_remshell_LDADD = src/pm/util/libmpiexec.la $(mpllib)
-src_pm_remshell_mpiexec_remshell_LDFLAGS = $(mpllibdir)
-EXTRA_src_pm_remshell_mpiexec_remshell_DEPENDENCIES = $(mpllib)
+src_pm_remshell_mpiexec_remshell_LDADD = src/pm/util/libmpiexec.la $(mpl_lib)
+EXTRA_src_pm_remshell_mpiexec_remshell_DEPENDENCIES = $(mpl_lib)
 # we may not want to add AM_CPPFLAGS for this program
 src_pm_remshell_mpiexec_remshell_CPPFLAGS = $(common_pm_includes) $(AM_CPPFLAGS)
 endif !PRIMARY_PM_REMSHELL

--- a/src/pm/util/Makefile.mk
+++ b/src/pm/util/Makefile.mk
@@ -25,9 +25,8 @@ src_pm_util_libmpiexec_la_CFLAGS = -g $(AM_CFLAGS)
 src_pm_util_libmpiexec_la_CPPFLAGS = $(common_pm_includes) $(AM_CPPFLAGS)
 
 # MPL
-src_pm_util_libmpiexec_la_LIBADD = $(mpllib)
-src_pm_util_libmpiexec_la_LDFLAGS = $(mpllibdir)
-EXTRA_src_pm_util_libmpiexec_la_DEPENDENCIES = $(mpllib)
+src_pm_util_libmpiexec_la_LIBADD = $(mpl_lib)
+EXTRA_src_pm_util_libmpiexec_la_DEPENDENCIES = $(mpl_lib)
 
 # We use the msg print routines (for now) - include these in the mpiexec
 # library so that we don't need to copy the source files

